application.py

```diff
@@ -1,6 +1,7 @@
 import os
```

Comment:
Yeah, I've noticed that bad markdown has the potential to ruin the page.
That being said, the old system had the same issue (just leave an unclosed <pre> tag or something like that)

Ah, I see what happened to the hint hiding.

Reply:
We will need to implement santization sometime in the near future, but since this is not really related to the markdown itself I'll open it as a new issue